File: QTD2.Infrastructure/Model/DIFSurveyEmployeeResponse/ImportData_DIFSurveyEmployeeResponse_VM.cs
using QTD2.Infrastructure.Model.Import_CSV_VM;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.DIFSurveyEmployeeResponse
{
    public class ImportData_DIFSurveyEmployeeResponse_VM : ImportData_VM
    {
        public int DifSurveyId { get; set; }
        public List<ImportDatum_DIFSurveyEmployeeResponse_VM> Data { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/DIFSurveyImport/ImportData_DIFSurveyEmployeeResponse_VM.ts
import { ImportData_VM } from "@models/ImportCSV/ImportData_VM";
import { ImportDatum_DIFSurveyEmployeeResponse_VM } from "./ImportDatum_DIFSurveyEmployeeResponse_VM";

export class ImportData_DIFSurveyEmployeeResponse_VM extends ImportData_VM{
    difSurveyId:string;
    data: ImportDatum_DIFSurveyEmployeeResponse_VM[];
}